src/test/java/task12/Task.java

```diff
@@ -0,0 +1,35 @@
+package task12;
+
+import java.util.Random;
+import java.util.Scanner;
+public class Task {
+    static void task1(){
+        Random rand = new Random();
```

Comment:
better to use full names for such variables

Reply:
Done